### PR TITLE
Clean and update FTShoes configuration files

### PR DESCRIPTION
**Pull request by lrapetti**
Following the update calibration matrix for PRO.01 (see https://github.com/robotology/forcetorque-yarp-devices/pull/23), I am updating also the configuration files in this repo.

I took the change to remove the unused and deprecated calibration matrices in the configuration files, and I have added a specific file for `PRO.01` shoes.

### CODE REVIEW

**Comment by Yeshasvitvs** [comment]
@lrapetti may be we can delete the following files as part of this cleanup:

- https://github.com/robotology/wearables/blob/clean-ftshoes-conf/app/xml/FTShoeLeftWearableDevice.xml
- https://github.com/robotology/wearables/blob/clean-ftshoes-conf/app/xml/FTShoeRightWearableDevice.xml
- https://github.com/robotology/wearables/blob/clean-ftshoes-conf/app/xml/FTShoesWearableDevice.xml
- https://github.com/robotology/wearables/blob/clean-ftshoes-conf/app/xml/FTShoesWearableDevice_2.xml

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - Added documentation for compiling the project in the `README.md` (https://github.com/robotology/wearables/pull/96)
 
+### Changed
+- FTShoes configuration files have been cleaned
```
Comment (Yeshasvitvs) [inline CHANGELOG.md:13]: Please add the Shoe model number

**Review by Yeshasvitvs — APPROVED** [review]
LGTM 👍🏾 
Just some minor comments

File: CHANGELOG.md (hunk 1 above)
Comment (lrapetti) [reply CHANGELOG.md]: Done

**Comment by lrapetti** [comment]
> @lrapetti may be we can delete the following files as part of this cleanup:
> 
> * https://github.com/robotology/wearables/blob/clean-ftshoes-conf/app/xml/FTShoeLeftWearableDevice.xml
> * https://github.com/robotology/wearables/blob/clean-ftshoes-conf/app/xml/FTShoeRightWearableDevice.xml
> * https://github.com/robotology/wearables/blob/clean-ftshoes-conf/app/xml/FTShoesWearableDevice.xml
> * https://github.com/robotology/wearables/blob/clean-ftshoes-conf/app/xml/FTShoesWearableDevice_2.xml

I would leave those file since they be convenient in case it is required to run a different pair of shoes, or in case it is required right/left shoe separate, or in case two pair of shoes run on the same network.

**Comment by Yeshasvitvs** [comment]
@lrapetti Merging. Thanks.

**Pull request merged**